pytorch_lightning/trainer/trainer.py

```diff
@@ -903,9 +903,6 @@ def run_pretrain_routine(self, model):
         # CORE TRAINING LOOP
         self.train()
 
-        # summarize profile results
```

Comment:
Does it mean we do not want to show the stat always?
Shall the `run_training_teardown()` be called also here?

Reply:
the profiler report is called within `train()`, i felt like that was the more logical place to find the code so i moved it into the `run_training_teardown()` method.